test/e2e_kubeadm/bootstrap_token_test.go

```diff
@@ -32,6 +32,9 @@ const (
 	bootstrapTokensAllowPostCSRClusterRoleName       = "system:node-bootstrapper"
 	bootstrapTokensCSRAutoApprovalClusterRoleBinding = "kubeadm:node-autoapprove-bootstrap"
 	bootstrapTokensCSRAutoApprovalClusterRoleName    = "system:certificates.k8s.io:certificatesigningrequests:nodeclient"
+	bootstrapTokensAuthExtraGroup                    = "auth-extra-groups"
```

Comment:
this is already defined here: https://github.com/kubernetes/kubernetes/blob/c179a9c9df4bfb6421258a5f31da9f595474ada4/staging/src/k8s.io/cluster-bootstrap/token/api/types.go#L57